app/Http/PathInfo.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace App\Http;
+
+/**
+ * syonetに用いるURLやパスを用いるクラス
+ */
+class PathInfo {
```

Comment:
__[code_sniffer]__
* Opening brace of a class must be on the line after the definition